### PR TITLE
AAP-13082: Remove community.general references

### PR DESCRIPTION
**Pull request by thom-at-redhat**
# Description

Remove community.general references
FIXES: AAP-13082

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [X] Refactor

## Checklist

- [X] Added changelog fragment
- [ ] Tests exist for affected features covering positive and negative scenarios

### CODE REVIEW

**Review by de1987 — APPROVED** [review]

**Review by de1987 — DISMISSED** [review]

**Review by de1987 — DISMISSED** [review]

**Comment by omaciel** [comment]
I'd recommend squashing the commits when you're done, but that's just my opinion :)

**Comment by thom-at-redhat** [comment]
@mergifyio squash

File: plugins/modules/rhsm_repo_info.py [hunk 1]
```diff
@@ -64,15 +64,17 @@ def rhsm_repo_info(module):
...
-                    "type": "yum-baseurl",
-                    "check_ssl": True if items['sslverify'] == '1' else False,
-                    "check_gpg": True if items['gpgcheck'] == '1' else False,
-                    "gpgkey_paths": items['gpgkey'],
-                    "state": 'present',
-                })
+                rhsm_info.append(
+                    {
+                        "name": repo,
+                        "base_url": items["baseurl"] if "baseurl" in items else None,
```
Comment (matoval) [inline plugins/modules/rhsm_repo_info.py:70]: What happens if base_url = None? Wouldn't that cause the image build to fail?

Comment (thom-at-redhat) [reply plugins/modules/rhsm_repo_info.py]: good idea! added specific error messages

**Pull request merged**